packages/fx-core/templates/core/v3Migration/js.ts.app.local.yml

```diff
@@ -106,7 +129,7 @@ deploy:
 
   {{/dotnetCommand}}
   {{#tab}}
-  - uses: file/updateEnv
+  - uses: file/updateEnv # Generate env to .env file
```

Comment:
For those with target `.env.teamsfx.local` or `.localSettings` or any target, prefer **Generate runtime environment variables** since the target is not TeamsFx context.

Reply:
Good suggestion, updated.